src/main/java/com/google/graphgeckos/dashboard/datatypes/Log.java

```diff
@@ -42,17 +50,31 @@ public Log(String[] logComponents) {
   }
 
   /**
-   * Returns the type of the log as a String. Can be null.
+   * @return the type of the log as a String. Can be null.
```

Comment:
nit: here and below, don't start a javadoc comment with a `@return` or `@param`; the previous version was OK: https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment

Reply:
but is it worth having these javadoc comments? https://google.github.io/styleguide/javaguide.html#s7.3.1-javadoc-exception-self-explanatory